pkg/controller/wip/controller_test.go

```diff
@@ -419,6 +419,8 @@ func TestReconcileInstanceWithParameters(t *testing.T) {
 
 	if e, a := v1alpha1.ConditionTrue, updateObject.Status.Conditions[0].Status; e != a {
 		t.Fatalf("Unexpected condition status: expected %v, got %v", e, a)
+		// temporary debugging
+		t.Fatalf("MORE: Unexpected condition status: expected %v, got %v : %+v", e, a, updateObject.Status.Conditions[0])
```

Comment:
if you think it would be helpful in the future, actually make this replacement. logging is cheap, debugging time is not.